docs/components/receive.md

```diff
@@ -108,29 +108,6 @@ Flags:
                                  TLS CA to verify clients against. If no client
                                  CA is specified, there is no client
                                  verification on server side. (tls.NoClientCert)
-      --remote-write.address="0.0.0.0:19291"
```

Comment:
Yes, we remove these however, where are we adding those? We need to generate documentation for router as well.